File: lab4/Java/Funcionario.java
public class Funcionario {
    protected String nomeFuncionario;
    protected float salarioFuncionario;

    public Funcionario(String nomeFuncionario, float salarioFuncionario) {
        this.nomeFuncionario = nomeFuncionario;
        this.salarioFuncionario = salarioFuncionario;
    }

    public String descricao(){
        return "Nome: " + nomeFuncionario + " - Salário: " + salarioFuncionario;
    }
}

File: lab4/C++/Funcionario.cpp
#include "Funcionario.hpp"

Funcionario::Funcionario(string nomeFuncionario, float salarioBase)
: nomeFuncionario(nomeFuncionario), salarioBase(salarioBase) {}

string Funcionario::descricao()
{
    return "Nome: " + nomeFuncionario + ", Salário: " + to_string(salarioBase);
}